cg/store/models.py

```diff
@@ -987,3 +988,59 @@ class Order(Base):
 
     def to_dict(self):
         return to_dict(model_instance=self)
+
+
+class DigitisationUnit(Base):
+    """Parent model representing the consumable units (cells/chips) on which the sample data is generated."""
+
+    __tablename__ = "digitisation_unit"
+
+    id: Mapped[PrimaryKeyInt]
+    type: Mapped[MachinesTypes]
+    model: Mapped[Str32]
+    internal_id: Mapped[UniqueStr]
+
+    batch_metrics: Mapped[list["BatchDigitisation"]] = orm.relationship(back_populates="unit")
+
+    __mapper_args__ = {
+        "polymorphic_on": "type",
+    }
+
+
+class BatchDigitisation(Base):
```

Comment:
Describes a "run" on the instrument. I see this as anything between starting the instrument, to spitting out BAM/FASTQ files. This can contain time stamps, performance metrics for the consumable used (item to the right). It should be noted that for ONT the flow cells can be run multiple times